hr_holidays_imposed_days/models/hr_employee.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, fields
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    def _get_search_imposed_parameters(self, employee):
+        """ """
+        res = [('company_id', '=', employee.company_id.id),
+               ('employee_ids', '=', False),  # no employee defined means all
+               ('date_from', '>=', fields.Datetime.now())
+               ]
+        return res
+
+    @api.model
+    @api.returns('self', lambda value: value.id)
+    def create(self, values):
+        """ add imposed days if a configuration exists """
+        imposed_holiday = self.env['hr.holidays.imposed']
+        holiday = self.env['hr.holidays']
+
+        emp = super(HrEmployee, self).create(values)
+
+        imposed = imposed_holiday.search(
+            self._get_search_imposed_parameters(emp)
+            )
+
+        if imposed:
```

Comment:
You can remove the `if` statement, as looping on an empty list works (it does nothing, so doesn't crash), but this is not blocking.